### ISSUE TITLE
[Android] Warning causes by namespace "${group}" since 2.x.x version

### ISSUE THREAD

**Issue by TomCorvus** [achorein/expo-share-intent#68]
**Describe the bug**
A warning appears on `pod install` since 2.x.x version 
`warn Invalid application's package name "${group}" in either 'AndroidManifest.xml' or 'build.gradle'. Read guidelines for setting the package name here: https://developer.android.com/studio/build/application-id`

It seems to be caused by:
```
android {
  def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION

  namespace "${group}"
  defaultConfig {
    versionCode 1
    versionName "${version}"
  }
}
```

in `build.gradle` 

I don't know why I see this during `pod install` btw.

- expo version (ex: 49) : 51
- package version (ex: 1.2.0) : 2.0.1
- platform target : Android / iOS

**Comment by achorein** [comment]
@TomCorvus i cannot reproduce with android studio jellyfish (Android 14 SDK 34, expo 51.0.17), is the problem persisting ?

**Comment by TomCorvus** [comment]
@achorein Yes
Add `"expo-share-intent": "^2.1.1"` in basic example and type `pod install` in ios folder.
<img width="909" alt="Capture d’écran 2024-07-10 à 09 18 57" src="https://github.com/achorein/expo-share-intent/assets/3451011/6f7c1602-b830-4d7a-966e-69301e93626a">

This message seems to not appear if expo-share-intent is called from the source folder. (./src/*) You need to get it as a node module.

**Comment by achorein** [comment]
fixed in v2.3.0
